Source/SpectrumAnalysis/KTChannelAggregator.cc

```diff
@@ -17,15 +19,22 @@ namespace Katydid
     KT_REGISTER_PROCESSOR(KTChannelAggregator, "channel-aggregator");
 
     KTChannelAggregator::KTChannelAggregator(const std::string& name) :
-            KTProcessor(name),
-            fSummedFrequencyData("agg-fft", this),
-            fPhaseChFrequencySumSlot("fft", this, &KTChannelAggregator::SumChannelVoltageWithPhase, &fSummedFrequencyData),
-            fActiveRadius(0.0516),
-            fNGrid(30),
-            fWavelength(0.0115),
-            fIsGridDefined(false),
-	    fUseAntiSpiralPhaseShifts(false),
-	    fAntiSpiralPhaseShifts()
+        KTProcessor(name),
+        fSummedFrequencyData("agg-fft", this),
+        fPhaseChFrequencySumSlot("fft", this, &KTChannelAggregator::SumChannelVoltageWithPhase, &fSummedFrequencyData),
+        fAxialSumSlot("ax-agg-fft", this, &KTChannelAggregator::SumChannelVoltageWithPhase, &fSummedFrequencyData),
+        fActiveRadius(0.0516),
```

Comment:
Values should be initialized to things that are valid values for those variables, but will generally _not_ be useful in an analysis.  This way all users are made to explicitly specify things in their configs and not rely on default values that might get changed. So, e.g. numbers of things initialized to 0; maxes and mins to 0 and 1, respectively; wavelength to 0.